changelog.json

```
@@ -1,5 +1,9 @@
 {
   "releases": {
+    "2.3.2-beta5": [
+      "[Fixed] Create issue in upstream repository for fork - #9232",
```

I think this should be "improved", and took a stab at more active phrasing
```suggestion
      "[Improved] Direct new issues created from a local fork to its upstream on GitHub - #9232",
```